workflows/simple_workflow.go

```diff
@@ -28,6 +29,10 @@ func (simpleWorkflow *SimpleWorkflow) GetNodes() []nodes.InterfaceNode {
 func (simpleWorkflow *SimpleWorkflow) runVirt(workflowContext *nodes.WorkflowContext) {
 	simpleWorkflow.Logger.Println("Running runVirt method.")
 	for _, node := range simpleWorkflow.GetNodes() {
-		node.Process(node, workflowContext)
+		var err error = simpleWorkflow.processNode(node, workflowContext)
```

Comment:
This error should bubble up 1 more level to the Workflow class. This is so that we can handle errors for all workflow types the same. I think instead of `break` we can replace it with a `return err`

Reply:
Makes sense, done.